src/screens/CarDetails.tsx

```diff
@@ -40,67 +44,164 @@ export default function CarDetails(props: CarDetailsProps) {
+                        {/* Car details */}
+                        <View style={{
+                            display: 'flex',
+                            alignItems: 'center',
+                            justifyContent: 'center',
+                            marginTop: "75%",
                         }}>
-                            {car.type}
-                        </H3>
-                    </View>
-                    {/* Move forward button */}
-                    <View style={{
-                        display: 'flex',
-                        alignItems: 'center',
-                        justifyContent: 'center',
-                        marginBottom: "15%"
-                    }}>
-                        <Button
-                            onPress={() => props.navigation.goBack()}
-                        >
-                            <H1 color="light" bold>Order car</H1>
-                            <View style={{ marginTop: 2 }}>
-                                <Image source={require('../../assets/right-arrow.png')} style={{ width: 26, height: 26, marginLeft: 10 }} />
-                            </View>
-                        </Button>
-                    </View>
+                            <H1 bold style={{ fontSize: 35 }}>
+                                {car.name}
+                            </H1>
+                            <H3 style={{
+                                marginTop: 10,
+                            }}>
+                                {car.type}
+                            </H3>
+                        </View>
+                        <Metrics car={car} />
+                        {/* Move forward button */}
+                        <View style={{
+                            display: 'flex',
+                            alignItems: 'center',
+                            justifyContent: 'center',
+                            marginBottom: "15%"
+                        }}>
+                            <Button
+                                onPress={() => props.navigation.goBack()}
+                            >
+                                <H1 color="light" bold>Order car</H1>
+                                <View style={{ marginTop: 2 }}>
+                                    <Image source={require('../../assets/right-arrow.png')} style={{ width: 26, height: 26, marginLeft: 10 }} />
+                                </View>
+                            </Button>
+                        </View>
+                    </View >
                 </View >
-            </View >
+                <View style={{
+                    position: "absolute",
+                    top: 0,
+                    left: 0,
+                }}>
+                    <Pressable onPress={() => props.navigation.goBack()}>
+                        <Image source={require('../../assets/right-arrow.png')} style={{ width: 30, height: 30, marginLeft: 15, marginTop: 15, transform: [{ rotate: "180deg" }] }} />
```

Comment:
Reeeeally don't like the rotate here.
Can we use something from `@expo/vector-icons` instead?

Reply:
This is a refactor element. There are multiple elements that could be moved to such SVG implemenntation